eventing/samples/github-events/README.md

```diff
@@ -0,0 +1,162 @@
+# GitHub Flow
```

Comment:
"Github Flow" --> "Creating a GitHub Flow Using Knative Eventing", or similar.

Reply:
How about "Reacting to GitHub Events"